### PR TITLE
Feature/add esco readme

### PR DESCRIPTION
**Pull request by tomtollafield**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,8 +22,46 @@ Click 'Add Graph', then 'Create a Local Graph'. Enter a graph name, set the pass
...
+1) Install the APOC plugin - go to plugins -> install
+2) Install the Neosemantics v4 plugin - https://github.com/neo4j-labs/neosemantics - copy jar to plugins directory, add to the end of neo4j.conf - dbms.unmanaged_extension_classes=n10s.endpoint=/rdf
+    NOTE: If any existing plugins have already set "dbms.unmanaged_extension_classes", combine the existing values with the required Neosemantics value, by comma separating them. (E.g. if both the Neosemantics and GraphQL plugins are installed, set the value to dbms.unmanaged_extension_classes=org.neo4j.graphql=/n10s.endpoint=/rdf)
+3) Restart graph instance
+4) Enable the browser setting "Enable multi statement query editor"
+5) NOTE: Use http://neo4j-labs.github.io/neosemantics/ for reference:
+    Using the desktop app - query window:
+    CREATE CONSTRAINT ON (n:Resource) ASSERT n.uri IS UNIQUE;
+    Add the namespace prefixes by calling :
+    call n10s.nsprefixes.add("adms","http://www.w3.org/ns/adms#");
```
Comment (lazcool) [inline README.md:36]: We don't need all these, and we need to create one for regulated-professions. Take a look [here](https://github.com/SkillsFundingAgency/dfc-servicetaxonomy-database/blob/master/Resources/Dockerfiles/Neo4jScripts/neo4j-db-config).

File: README.md [hunk 2]
```diff
@@ -22,8 +22,46 @@ Click 'Add Graph', then 'Create a Local Graph'. Enter a graph name, set the pass
...
+    call n10s.nsprefixes.add("esco","http://data.europa.eu/esco/model#");
+    call n10s.nsprefixes.add("rov","http://www.w3.org/ns/regorg#");
+    call n10s.nsprefixes.add("dcat","http://www.w3.org/ns/dcat#");
+    call n10s.nsprefixes.add("euvoc","http://publications.europa.eu/ontology/euvoc#");
+    call n10s.nsprefixes.add("prov","http://www.w3.org/ns/prov#");
+    call n10s.nsprefixes.add("foaf","http://xmlns.com/foaf/0.1/");
+    call n10s.nsprefixes.add("qdr","http://data.europa.eu/esco/qdr#");
+    call n10s.nsprefixes.add("ncs","http://nationalcareers.service.gov.uk/taxonomy#");
+6) Import the ESCO data using this command (replacing the path to the file appropriately):
+    CALL n10s.graphconfig.init();
```
Comment (lazcool) [inline README.md:56]: We need to pass some params to init(), see [here](https://github.com/SkillsFundingAgency/dfc-servicetaxonomy-database/blob/master/Resources/Dockerfiles/Neo4jScripts/neo4j-db-config).

File: README.md [hunk 3]
```diff
@@ -22,8 +22,46 @@ Click 'Add Graph', then 'Create a Local Graph'. Enter a graph name, set the pass
...
+    CALL semantics.importRDF("file:///Users/wayne.local/Downloads/esco_v1.0.3.ttl","Turtle", { handleMultival: 'ARRAY', multivalPropList : ['http://www.w3.org/2004/02/skos/core#altLabel', 'http://www.w3.org/2004/02/skos/core#hiddenLabel'], languageFilter: "en" })
```
Comment (lazcool) [inline README.md:57]: This should be import.fetch, see [here](https://github.com/SkillsFundingAgency/dfc-servicetaxonomy-database/blob/master/Resources/Dockerfiles/Neo4jScripts/neo4j-import).

File: README.md [hunk 4]
```diff
@@ -22,8 +22,46 @@ Click 'Add Graph', then 'Create a Local Graph'. Enter a graph name, set the pass
...
+#### Loading ESCO data
+
+1) Install the APOC plugin - go to plugins -> install
+2) Install the Neosemantics v4 plugin - https://github.com/neo4j-labs/neosemantics - copy jar to plugins directory, add to the end of neo4j.conf - dbms.unmanaged_extension_classes=n10s.endpoint=/rdf
+    NOTE: If any existing plugins have already set "dbms.unmanaged_extension_classes", combine the existing values with the required Neosemantics value, by comma separating them. (E.g. if both the Neosemantics and GraphQL plugins are installed, set the value to dbms.unmanaged_extension_classes=org.neo4j.graphql=/n10s.endpoint=/rdf)
+3) Restart graph instance
+4) Enable the browser setting "Enable multi statement query editor"
+5) NOTE: Use http://neo4j-labs.github.io/neosemantics/ for reference:
+    Using the desktop app - query window:
+    CREATE CONSTRAINT ON (n:Resource) ASSERT n.uri IS UNIQUE;
```
Comment (lazcool) [inline README.md]: Could you convert this to a code block with comments, so it can be more easily cut and pasted?

File: README.md [hunk 5]
```diff
@@ -22,8 +22,46 @@ Click 'Add Graph', then 'Create a Local Graph'. Enter a graph name, set the pass
 
 That's all you need to do for syncing within the editor to work. To perform interactive queries against the graph, once the graph is Active, click 'Manage' and then on the next page, click 'Open Browser'. If you're unfamiliar with the browser or Neo4j in general, check out the [docs](https://neo4j.com/developer/neo4j-browser/).
 
-Optional Steps:
-To link data to ESCO you will need to install the ESCO TTL file into Neo4J, there are a number of steps that need to be followed and executed, for more information on how to do this view [Neo4J Setup](https://skillsfundingagency.atlassian.net/wiki/spaces/DFC/pages/1491501074/SPIKE+RDF+hosting+-+Neo4j+cluster) on Confluence. (This link is only available to Internal users)
+#### Loading ESCO data
+
+1) Install the APOC plugin - go to plugins -> install
+2) Install the Neosemantics v4 plugin - https://github.com/neo4j-labs/neosemantics - copy jar to plugins directory, add to the end of neo4j.conf - dbms.unmanaged_extension_classes=n10s.endpoint=/rdf
```
Comment (lazcool) [inline README.md:28]: Don't think this is needed?
We should add the unrestricted and whitelist properties though, see [here](https://github.com/SkillsFundingAgency/dfc-servicetaxonomy-database/blob/master/Resources/Dockerfiles/neo4jescodb.Dockerfile).

Comment (lazcool) [inline README.md:28]: Don't forget your analyzer plugin 😄

File: README.md [hunk 6]
```diff
@@ -22,8 +22,29 @@ Click 'Add Graph', then 'Create a Local Graph'. Enter a graph name, set the pass
 
 That's all you need to do for syncing within the editor to work. To perform interactive queries against the graph, once the graph is Active, click 'Manage' and then on the next page, click 'Open Browser'. If you're unfamiliar with the browser or Neo4j in general, check out the [docs](https://neo4j.com/developer/neo4j-browser/).
 
-Optional Steps:
-To link data to ESCO you will need to install the ESCO TTL file into Neo4J, there are a number of steps that need to be followed and executed, for more information on how to do this view [Neo4J Setup](https://skillsfundingagency.atlassian.net/wiki/spaces/DFC/pages/1491501074/SPIKE+RDF+hosting+-+Neo4j+cluster) on Confluence. (This link is only available to Internal users)
+#### Loading ESCO data
```
Comment (lazcool) [inline README.md:25]: Could you add a link to where the ESCO data can be downloaded?

File: README.md (hunk 5 above)
Comment (tomtollafield) [reply README.md:28]: It was in the existing guide and on the neosemantics docs it says to add it:
image

Comment (tomtollafield) [reply README.md:28]: Synonyms is currently in another section further down, shall I move it up?

Comment (lazcool) [reply README.md:28]: That's interesting.
We'll have to dig a bit on this setting, as we might have to add it to our neo dockerfile.

Comment (lazcool) [reply README.md:28]: Yeah, I think it makes sense to have it here.

**Pull request merged**